go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/onsi/ginkgo v1.14.0
 	github.com/onsi/gomega v1.10.1
 	github.com/pkg/errors v0.9.1
-	github.com/submariner-io/admiral v0.6.0
+	github.com/submariner-io/admiral v0.6.1-0.20200903074731-a69b58d4bceb
```

Comment:
This was bumped to v0.6.1 in another PR.

Reply:
Changed version.